hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestObserverWithRouter.java

```diff
@@ -599,9 +604,6 @@ public void testRouterResponseHeaderState() {
     namespaceIdMap.put("ns2", new LongAccumulator(Math::max, Long.MIN_VALUE));
 
     Map<String, Long> mockMapping = new HashMap<>();
-    mockMapping.put("ns0", 10L);
-    mockMapping.put("ns2", 100L);
-    mockMapping.put("ns3", Long.MIN_VALUE);
```

Comment:
Var mockMapping should be removed . Because routerStateIdContext.updateResponseState(responseHeaderBuilder) will update RouterFederatedState  with namespaceIdMap's keys and values that meet the requirements. 
Thanks